src/reducers/index.js

```diff
@@ -0,0 +1,50 @@
+//import { ADD_FEATURES, REMOVE_FEATURES } from '../actions/index';
+
+const initialState = {
+    additionalPrice: 0,
```

Comment:
I see that you're changing the price of the car inside the car object.  It's a good attempt to change the price, but I think that was actually supposed to represent the base price of the car.  The additionalPrice (the first property of the initialState object is where you should be adding the additional feature's prices